Modules/IO/PhilipsREC/src/itkPhilipsRECImageIO.cxx

```diff
@@ -190,18 +190,16 @@ PhilipsRECImageIOGetImageTypeOffset(int
                                     PhilipsPAR::PARSliceIndexImageTypeVector    sliceImageTypesIndex,
                                     PhilipsPAR::PARSliceIndexScanSequenceVector sliceScanSequenceIndex)
 {
-  int index = volumeIndex * parParam.num_slice_repetitions * numSlices + slice * parParam.num_slice_repetitions;
-  int i;
-
-  for (i = 0; i < parParam.num_slice_repetitions; ++i)
+  const int index = volumeIndex * parParam.num_slice_repetitions * numSlices + slice * parParam.num_slice_repetitions;
+  for (int image_type_offset = 0; image_type_offset < parParam.num_slice_repetitions; ++image_type_offset)
   {
-    if ((sliceImageTypesIndex[index + i].second == imageType) &&
-        (sliceScanSequenceIndex[index + i].second == scanSequence))
+    if ((sliceImageTypesIndex[index + image_type_offset].second == imageType) &&
+        (sliceScanSequenceIndex[index + image_type_offset].second == scanSequence))
     {
-      break;
+      return image_type_offset;
     }
   }
-  return i;
+  return parParam.num_slice_repetitions;
```

Comment:
Thank you for addressing my concern about a possible behavior change of `PhilipsRECImageIOGetImageTypeOffset` (at https://github.com/InsightSoftwareConsortium/ITK/pull/4977#discussion_r1854365226). Looks fine 👍

Why do you now change the name of the identifier, from `i` to `image_type_offset`?

Personally I'd rather have the renaming of identifiers in a separate PR, if necessary. This PR is already big enough to me. But anyway, if you do want to rename the variable, shouldn't it be something like `imageTypeOffset`, in camel case?

If there is a general agreement that we use _both_ `camelCase` _and_ `snake_case` for local variables, no problem to me. I'm just asking!

Reply:
Reverted variable to "i".